Fix build failure:
    recv.c:17: /usr/include/linux/if_arp.h:114:8: error: redefinition of 'struct
on modern linux-headers.

Patch by Murilo Morais
Bug: https://bugs.gentoo.org/625236
--- a/includes.h	2015-11-19 10:58:52.000000000 -0300
+++ b/includes.h	2017-07-15 09:32:17.024877205 -0300
@@ -86,7 +86,7 @@
 #include <net/if_types.h>
 #endif
 
-#if defined(HAVE_NET_IF_ARP_H) && !defined(ARPHRD_ETHER)
+#if (defined(HAVE_NET_IF_ARP_H) && !defined(ARPHRD_ETHER)) || HAVE_LINUX_IF_ARP_H
 #include <net/if_arp.h>
 #endif				/* defined(HAVE_NET_IF_ARP_H) && !defined(ARPHRD_ETHER) */
 
@@ -101,8 +101,3 @@
 #ifdef HAVE_IFADDRS_H
 #include <ifaddrs.h>
 #endif
-
-#ifdef HAVE_LINUX_IF_ARP_H
-#include <linux/if_arp.h>
-#endif
-
